module/VuFind/src/VuFind/Cache/Manager.php

```diff
@@ -354,24 +354,16 @@ protected function createNoCache($cacheName)
     }
 
     /**
-     * Add a file cache to the manager and ensure that necessary directory exists.
+     * Ensure that a cache directory exists.
      *
-     * @param string $cacheName    Name of new cache to create
      * @param string $dirName      Directory to use for storage
      * @param array  $overrideOpts Options to override default values.
      *
      * @return void
      */
-    protected function createFileCache($cacheName, $dirName, $overrideOpts = [])
+    public function ensureCacheDirectoyExists($dirName, $overrideOpts = [])
```

Comment:
Typo in method name:

```suggestion
    public function ensureCacheDirectoryExists($dirName, $overrideOpts = [])
```

Reply:
Thanks for catching this!